tests/Data/AlphaNumTest.php

```diff
@@ -35,14 +35,15 @@ public static function stringValidateProvider():array{
 			['ABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890 $%*+-./:', true],
 			['abc', false],
 			['ÄÖÜ', false],
-			[',', true],
+			[',', false],
```

Comment:
I think perhaps the tests should actually try to generate a QR with this data too. As it will crash in AlphaNum::ord(). Would have revealed this bug earlier.